src/utils/__tests__/textUtils.spec.js

```diff
@@ -45,10 +45,15 @@ describe('Text', () => {
       expect(result).toEqual([{string: 'uilib', shouldHighlight: false}, {string: '@wix', shouldHighlight: true}, {string: '.com', shouldHighlight: false}]);
     });
 
-    it('Should handle empty string .', () => {
+    it('Should handle empty string.', () => {
       const result = getTextPartsByHighlight('@ancing in the @ark', '');
       expect(result).toEqual([{string: '@ancing in the @ark', shouldHighlight: false}]);
     });
+
+    it('Should handle full string.', () => {
+      const result = getTextPartsByHighlight('Dancing in the Dark', 'Dancing in the Dark');
+      expect(result).toEqual([{string: 'Dancing in the Dark', shouldHighlight: true}]);
+    });
```

Comment:
Consider adding a test for more than full string: 
`getTextPartsByHighlight('Dancing in the Dark', 'Dancing in the Darkk');`

Reply:
I don't think this is very important in the current logic, adding in case it ever changes 👍 
Also noticed we don't have these tests for the array --> found another bug.